tests/integration_tests/test_integration_elasticsearchstore.py

```diff
@@ -219,3 +180,343 @@ def test_list_all_columns_with_fake_experiment_id(init_store):
                                tags=[])
     actual_columns = init_store.list_all_columns("fake_id", ViewType.ALL)
     assert expected_columns.__dict__ == actual_columns.__dict__
+
+
+def test_get_run(init_store):
+    expected_run_info = RunInfo(run_uuid="7b2e71956f3d4c08b042624a8d83700d",
+                                experiment_id="hTb553MBNoOYfhXjnnQh",
+                                user_id="1",
+                                status="RUNNING",
+                                start_time=1597324762662,
+                                end_time=None,
+                                lifecycle_stage="active",
+                                artifact_uri="artifact_path/7b2e71956f3d4c08b042624a8d83700d"
+                                "/artifacts",
+                                run_id="7b2e71956f3d4c08b042624a8d83700d")
+
+    expected_metrics = [Metric(key="metric0", value=15.0, timestamp=1597324762700, step=0),
+                        Metric(key="metric0", value=7.0, timestamp=1597324762742, step=1),
+                        Metric(key="metric0", value=20.0, timestamp=1597324762778, step=2),
+                        Metric(key="metric1", value=20.0, timestamp=1597324762815, step=0),
+                        Metric(key="metric1", value=0.0, timestamp=1597324762847, step=1),
+                        Metric(key="metric1", value=7.0, timestamp=1597324762890, step=2)]
+
+    expected_params = [Param(key="param0", value="val2"),
+                       Param(key="param1", value="Val1"),
+                       Param(key="param2", value="Val1"),
+                       Param(key="param3", value="valeur4")]
+
+    expected_tags = [RunTag(key="tag0", value="val2"),
+                     RunTag(key="tag1", value="test3"),
+                     RunTag(key="tag2", value="val2"),
+                     RunTag(key="tag3", value="test3")]
+
+    expected_run_data = RunData(metrics=expected_metrics,
+                                params=expected_params, tags=expected_tags)
+
+    expected_run = Run(run_info=expected_run_info, run_data=expected_run_data)
```

Comment:
You don't need to create these objects, you can directly check with the objects you use to create these